solr/solr-ref-guide/modules/deployment-guide/pages/solr-in-docker.adoc

```diff
@@ -127,7 +127,7 @@ So, typically you will use the `solr-precreate` command which prepares the speci
 docker run -d -p 8983:8983 --name my_solr solr solr-precreate gettingstarted
 ----
 
-The `solr-precreate` command takes an optional extra argument to specify a configset directory below `/opt/solr/server/solr/configsets/` or you can specify a full path to a custom configset inside the container:
+The `solr-precreate` command takes an optional extra argument to specify a configset directory below `/var/solr/data/configsets/` or you can specify a full path to a custom configset inside the container:
```

Comment:
The `configsets` are not moved, they are still part of the (read-only) location as before. You can verfity this by a simple docker run command:
```bash
docker run --rm solr ls -la /var/solr/data
total 8
drwxrwx--- 2 solr solr 4096 Jan 13 13:54 .
drwxrwx--- 4 solr root 4096 Jan 13 13:54 ..
```

```bash
docker run --rm solr ls -l /opt/solr/server/solr/configsets
total 8
drwxr-xr-x 3 root root 4096 Nov 11 07:31 _default
drwxr-xr-x 3 root root 4096 Nov 11 07:31 sample_techproducts_configs
```